.github/workflows/manual-release.yml

```
@@ -0,0 +1,35 @@
+        required: true
+      prerelease:
+        description: 'Prerelease?'
+        required: false
+        default: 'false'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Download Artifacts
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)